bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/manager/ShellyManagerPage.java

```diff
@@ -321,7 +321,7 @@ protected Map<String, String> fillProperties(Map<String, String> properties, Str
                 break;
             case OFFLINE:
                 ThingStatusDetail sd = th.getThing().getStatusInfo().getStatusDetail();
-                if (uid.contains(THING_TYPE_SHELLYUNKNOWN_STR) || (sd == ThingStatusDetail.CONFIGURATION_ERROR)
+                if (uid.contains(THING_TYPE_SHELLYUNKNOWN.getId()) || (sd == ThingStatusDetail.CONFIGURATION_ERROR)
```

Comment:
It is unclear for me what `uid` is, but at least when called from here:

https://github.com/openhab/openhab-addons/blob/278d371326694953eb5bf5eda182538a9b79f87d/bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/manager/ShellyManagerOverviewPage.java#L90

it seems to be `ThingUID`, not `ThingTypeUID`, so I don't think this will work.